tests/virt/upgrade/utils.py

```diff
@@ -193,5 +198,29 @@ def vm_is_not_migrateable(vm):
     )
     if DataVolume.AccessMode.RWO in vm_access_modes:
         LOGGER.info(f"Cannot migrate a VM {vm.name} with RWO PVC.")
-        return True
-    return False
+        return False
+    return True
+
+
+def verify_vms_boot_time(vm_list, initial_boot_time):
```

Comment:
```suggestion
def verify_linux_vms_boot_time(vm_list, initial_boot_time):
```

Reply:
`verify_linux_boot_time` to match windows function below